<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegWindow</class>
 <widget class="QWidget" name="RegWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>172</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>172</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="3">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="m_lblR12">
       <property name="text">
        <string>R48</string>
       </property>
       <property name="buddy">
        <cstring>m_pReg4</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="m_pReg12"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="m_lblR13">
       <property name="text">
        <string>R52</string>
       </property>
       <property name="buddy">
        <cstring>m_pReg5</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="m_pReg13"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="m_lblR14">
       <property name="text">
        <string>R56</string>
       </property>
       <property name="buddy">
        <cstring>m_pReg6</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="m_pReg14"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="m_lblR15">
       <property name="text">
        <string>R60</string>
       </property>
       <property name="buddy">
        <cstring>m_pReg7</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="m_pReg15"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QLabel" name="m_pFlagRegister">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="m_lblR0">
       <property name="text">
        <string>R0</string>
       </property>
       <property name="buddy">
        <cstring>m_pReg0</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="m_pReg0"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="m_lblR1">
       <property name="text">
        <string>R4</string>
       </property>
       <property name="buddy">
        <cstring>m_pReg1</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="m_pReg1"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="m_lblR2">
       <property name="text">
        <string>R8</string>
       </property>
       <property name="buddy">
        <cstring>m_pReg2</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="m_pReg2"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="m_lblR3">
       <property name="text">
        <string>R12</string>
       </property>
       <property name="buddy">
        <cstring>m_pReg3</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="m_pReg3"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QPushButton" name="m_pSubmit">
     <property name="text">
      <string>Set this state</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QLabel" name="m_lblR9">
       <property name="text">
        <string>R36</string>
       </property>
       <property name="buddy">
        <cstring>m_pReg5</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="m_lblR10">
       <property name="text">
        <string>R40</string>
       </property>
       <property name="buddy">
        <cstring>m_pReg6</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="m_pReg10"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="m_lblR11">
       <property name="text">
        <string>R44</string>
       </property>
       <property name="buddy">
        <cstring>m_pReg7</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="m_pReg8"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="m_pReg9"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="m_pReg11"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="m_lblR8">
       <property name="text">
        <string>R32</string>
       </property>
       <property name="buddy">
        <cstring>m_pReg4</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="m_lblR4">
       <property name="text">
        <string>R16</string>
       </property>
       <property name="buddy">
        <cstring>m_pReg4</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="m_pReg4"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="m_lblR5">
       <property name="text">
        <string>R20</string>
       </property>
       <property name="buddy">
        <cstring>m_pReg5</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="m_pReg5"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="m_lblR6">
       <property name="text">
        <string>R24</string>
       </property>
       <property name="buddy">
        <cstring>m_pReg6</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="m_pReg6"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="m_lblR7">
       <property name="text">
        <string>R28</string>
       </property>
       <property name="buddy">
        <cstring>m_pReg7</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="m_pReg7"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
